src/controller/motor.c

```diff
@@ -657,12 +643,12 @@ void TIM1_CAP_COM_IRQHandler(void) __interrupt(TIM1_CAP_COM_IRQHANDLER)
   }
   else // nothing to limit, so adjust duty_cycle to duty_cycle_target, including ramping
   {
-    // if brake is active, keep duty cycle target at 0
-    if (ui8_motor_controller_state & MOTOR_CONTROLLER_STATE_BRAKE) { ui8_controller_duty_cycle_target = 0; }
-    
     if (ui8_controller_duty_cycle_target > ui8_g_duty_cycle)
     {
-      if (ui16_counter_duty_cycle_ramp_up++ >= ui16_duty_cycle_ramp_up_inverse_step)
+      // limit duty cycle ramp up
+      if (ui16_duty_cycle_ramp_up_inverse_step < 20 ) { ui16_duty_cycle_ramp_up_inverse_step = 20; }
```

Comment:
Huh?
You've deliberately increased the define that sets this variable to 60, and now you're capping it at 20?

Reply:
If it is less than 20 it will be set to 20. It is correct. Note that it is the inverse step.

Reply:
Ah sorry, my bad, misread the comparison :)

Reply:
No problem at all!